Fix documentation on retry-all vs upload-retry (#3898)
transloadit/uppy: website/src/docs/core.md

### `upload-retry`

> Fired when an upload has been retried (after an error, for example). This is not fired when the user click on retry-all.
> 
> > ⚠️ Note that this method is event is not triggered when the user retry all uploads. (it trigger a retry-all event)

Reviewer: ```suggestion
Fired when an upload has been retried (after an error, for example).

> ⚠️ Note that this method is event is not triggered when the user retries all uploads, it will trigger the `retry-all` event intstead.
```

Fired when an upload has been retried (after an error, for example).

> > ⚠️ Note that this method is event is not triggered when the user retries all uploads, it will trigger the `retry-all` event intstead.

Reviewer: ```suggestion
> ⚠️ This event is not triggered when the user retries all uploads, it will trigger the `retry-all` event instead.
```
